services/api/src/utils/middleware/error-handler.js

```diff
@@ -6,14 +6,17 @@ async function errorHandler(ctx, next) {
   try {
     await next();
   } catch (err) {
-    let { status, message, details } = err;
+    let { status = 500, message, details, expose } = err;
```

Comment:
I guess the line below overriding to 500 can go away and this can be `const` instead then

Reply:
the issue is the `message` field